contracts/ResultComputation.sol

```diff
@@ -0,0 +1,120 @@
+pragma solidity ^0.5.0;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+/**
+ * @title ResultComputation
+ * @dev Computes the results of voting on a data value. The voted values, along with their weights, are kept in a sorted
+ * linked list as votes come in. The result is the weighted median of the values, computed by iterating over the list
+ * when needed.
+ * TODO(ptare): All methods are internal because we don't have a migration system written yet, so libraries with public
```

Comment:
Note: since the library is mostly for organizational purposes, I don't see this as something we need to do unless we run out of space in the main contract. Making them external will really just increase gas costs and won't really buy us anything other than space. To get library upgradability, we would need to use something more sophisticated than a library since its address will be hardcoded in the binary.